problem2.py

```diff
@@ -0,0 +1,34 @@
+""" 
+Time Complexity : O(N) N= no.of nodes
+Space Complexity :O(m) size of queue
```

Comment:
So what can be the maximum size of the queue in worst case? It will be all the leaf nodes. Which means SC = O(n) where n is the no. of nodes